_maps/map_files/KiloStation/KiloStation.dmm

```diff
@@ -51619,7 +51619,7 @@
 /obj/structure/flora/bush/lavendergrass/style_random,
 /obj/structure/flora/bush/fullgrass/style_random,
 /obj/structure/flora/bush/grassy/style_random,
-/mob/living/simple_animal/butterfly,
+/mob/living/basic/butterfly,
```

Comment:
make sure to include one of those path rename scripts

Reply:
note to self: we should lint for this at some point

Reply:
Look for a change in paths, then flag with a "Needs UpdatePaths" label if map edits are also made?

Reply:
Yes something like that

Reply:
theoretically we can also unit test to make sure that the path update tools when run over your maps dont cause any changes. dunno how often that regresses

Reply:
we lint for this it's called the map doesn't compile 😎